### DISCUSSION TITLE
Anwenderfrage: Zeitungsausgaben ohne Datumsangabe ; wie Ausgabenvorgänge anlegen?

### DISCUSSION THREAD

**Discussion by SonjaGellert** [kitodo/kitodo-production#6263] [General]
Auch wenn das Problem bei uns noch nicht akut ist, wollte ich heute mal eine relativ allgemeine Frage an die Community stellen.

Wir sind aktuell dabei, einige alte/saure Zeitungsbestände zu scannen und wollen das alles in Zukunft mit Kitodo verwalten. Als Anwender habe ich für mich festgestellt, das Kitodo für die Verarbeitung von Zeitungsausgaben zwingend ein Datum braucht, weswegen das Datum direkt am Anfang beim Anlegen der einzelnen Zeitungsausgabenvorgänge ja auch abgefragt wird.
Das geht mit dem Kalendermodul sehr komfortabel und ist hier nicht das Problem.

Mein Problem taucht auf, wenn eine Zeitung kein Datum vermerkt hat, sondern nur eine Heft/Ausgaben Zählung und vielleicht ein Jahr und Monat.
Das ist nach meiner Einschätzung etwa bei 50% unserer Zeitungen der Fall.

Im Moment nutzen wir die Ausgabenzählung dann als Tag und fingieren daraus ein Datum. Beispielsweise. Die erste Ausgabe im Januar 1923, wird 1923-01-01
die zweite Ausgabe im Januar: 1923-01-02.
die dritte Ausgabe, die dann im Februar erschien: 1923-02-03
Das ist aber sehr unpraktisch bei der Nutzung des Kalendermodus.

Wir können doch nicht der einzige sein, der Kitodo anwendet, der keine Zeitungsausgaben ohne genaues Datum hat?
Wie legen Sie in Ihrer Einrichtung Zeitungen ohne genaue Datumsangabe an?

**Comment by apiller** [comment]
Wir fingieren auch ein Datum, wenn kein konkretes Datum genannt ist. Üblicherweise dann auch der 1. des Monats. Wenn es zwei Ausgaben pro Monat gibt liegt die 2. Ausgabe dann aber eher auf dem 15. 
Teilweise lassen sich im Impressum bzw. im Katalogisat auch Hinweise finden, wann die Ausgaben üblicherweise erschienen sind. 
Das wir ein Datum fingieren müssen ist aber eher die Ausnahme. Bei uns sind es dann eher Zeitschriften, wenn kein Datum vorliegt und die laufen  über ein anderes Projekt.

**Comment by andre-hohmann** [comment]
Ich kann mich @apiller weitgehend anschließen. 
Wenn es allerdings keine Hinweise auf ein Datum gibt, erstellen wir am ersten des Monats auch mehrere Ausgaben.
Um Missverständnisse zu vermeiden erfassen wir intern normierte Titel, so dass in der Kalenderansicht rasch erkennbar ist, dass es sich um ein geschätztes Datum handelt. 

Beispiele finden sich zum Beispiel unter: 
- [Leipziger Monatsschrift für Textil-Industrie -1908](https://digital.slub-dresden.de/oai/?verb=GetRecord&metadataPrefix=mets&identifier=oai:de:slub-dresden:db:id-1679165712-1908)
- In der Listenansicht ist besser ersichtlich, wenn zwei Ausgaben am Tag erschienen sind
- In diesem Fall haben wir allerdings häufig das Datum des Redaktionsschlusses angewendet. 

In folgendem Beispiel haben wir Ausgaben auch auf den ersten der Woche datiert und entsprechend gekennzeichnet: 
- [Friedrich Georg Wieck's deutsche illustrirte Gewerbezeitung - 1861](https://digital.slub-dresden.de/oai/?verb=GetRecord&metadataPrefix=mets&identifier=oai:de:slub-dresden:db:id-1679159151-1861)
- In der Listenansicht ist die Bezeichnung der Ausgaben besser ersichtlich

**Comment by SonjaGellert** [comment]
Danke für die Rückmeldung @apiller @andre-hohmann 

Es beruhigt mich wirklich, das wir nicht allein sind und mit fingierten Daten arbeiten. Die oben beschriebene Lösung haben wir angefangen, um es den Kolleg*innen am Scanner einfacher bei der Erfassung zu machen.

Wahrscheinlich bleibt aber der Schritt nicht aus, nach dem Scanvorgang, die erstellten Dateien/Ordner vor dem Kitodo Import noch einmal anzupassen, um eine gewisse regelmäßige Laufzeit zu fingieren.

Immerhin weiß ich nach dem Scan der Zeitung schon einmal, wie viele Ausgaben pro Woche/Monat erschienen sind.
